docs/api.md

```diff
@@ -183,12 +183,14 @@ is not restricted to the official set if [`download.mirror`](#download) is set.
 *Boolean* or *Object* (default: `false`)
 
 Whether to package the application's source code into an archive, using [Electron's archive format](https://github.com/electron/asar). Reasons why you may want to enable this feature are described in [an application packaging tutorial in Electron's documentation](http://electron.atom.io/docs/v0.36.0/tutorial/application-packaging/). When the value is `true`, pass default configuration to the `asar` module. The configuration values listed below can be customized when the value is an `Object`. Supported parameters include, but are not limited to:
+- `filename` (*String*): circumvents the asar generation and lets you specify your own pre-built asar file/directory.
 - `ordering` (*String*): A path to an ordering file for packing files. An explanation can be found on the [Atom issue tracker](https://github.com/atom/atom/issues/10163).
 - `unpack` (*String*): A [glob expression](https://github.com/isaacs/minimatch#features), when specified, unpacks the file with matching names to the `app.asar.unpacked` directory.
 - `unpackDir` (*String*): Unpacks the dir to the `app.asar.unpacked` directory whose names exactly or pattern match this string. The `asar.unpackDir` is relative to `dir`.
 
   Some examples:
 
+  - `asar.filename = './dist/app.asar'` will short-circuit creating the asar, and just copy that one into the electron bundle.
```

Comment:
I don't think there needs to be an example for this, it should be self-explanatory.